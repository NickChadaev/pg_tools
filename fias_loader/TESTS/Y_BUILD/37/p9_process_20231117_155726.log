2023-11-17 15:57:26 home/n.chadaev@abrr.local
127.0.0.1 5432 unsi_test_37 postgres /home/n.chadaev@abrr.local/Y_BUILD/37/stage_gar_c_37.csv 2023-11-13
--------------------------------------------------------
[15:57:26] ... Ивановская обл., id_region: 64, Execute: /home/n.chadaev@abrr.local/Y_BUILD/37/stage_gar_c_37.csv, process_id:25345
[15:57:26] ... Start process
[15:57:26] ...  -- Version
[15:57:26] ...  -- Очистка от данных
[15:57:26] ...  -- Убираем индексы
[15:57:26] ...  -- Set UNLOGGED
[15:57:26] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_APARTMENT_TYPES_20231113_78e8cff6-619c-4b2e-806f-6c3c18c41d8c.XML
[15:57:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ADDR_OBJ_TYPES_20231113_15f2ec37-c925-4252-b49a-5473d975a16f.XML
[15:57:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ROOM_TYPES_20231113_32952955-cd74-40c1-bc84-06d94fe9559d.XML
[15:57:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_OPERATION_TYPES_20231113_5151274c-91f4-4286-aaff-fc37112b660a.XML
[15:57:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_PARAM_TYPES_20231113_fe767969-20f3-4044-9ef6-67b4cb2881ce.XML
[15:57:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_HOUSE_TYPES_20231113_507d1000-4460-4d2f-a5b1-2ab5b9cd8a26.XML
[15:57:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ADDHOUSE_TYPES_20231113_d99b31fe-7b05-400a-ac92-23919e58da07.XML
[15:57:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_OBJECT_LEVELS_20231113_b5fa7b3d-ea26-40c9-8b3a-9b986261ea66.XML
[15:57:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_NORMATIVE_DOCS_TYPES_20231113_b8068ed2-dd18-4365-abb7-3dbd98698c35.XML
[15:57:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_NORMATIVE_DOCS_KINDS_20231113_2f0f38c6-d598-4fd7-b325-2aa6aca29ed2.XML
[15:57:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/37/AS_REESTR_OBJECTS_20231113_837258f2-1d59-4d07-85a7-99549e4cf9bf.XML
[16:07:30] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/37/AS_ADDR_OBJ_20231113_f4494f2b-e9b6-4f37-92d4-f29164dc9d0f.XML
[16:07:45] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/37/AS_HOUSES_20231113_1eaf2933-44e9-403f-bda8-23b74277b127.XML
[16:17:56] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/37/AS_STEADS_20231113_3332be81-d9af-4cc0-bfff-d5533f773f96.XML
[16:19:07] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/37/AS_APARTMENTS_20231113_330a6056-815d-4af5-a68d-c5c91e649a2f.XML
[16:23:25] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/37/AS_ROOMS_20231113_ceea8fd1-2068-4539-9499-4c863fbfedbf.XML
[16:23:30] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/37/AS_CARPLACES_20231113_c1f644e8-cbd0-46e3-b083-ac8c70970f6e.XML
[16:23:31] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/37/AS_ADM_HIERARCHY_20231113_73b051fb-324d-4913-8219-9e79c2e59e14.XML
[16:37:56] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/37/AS_MUN_HIERARCHY_20231113_a5dc8ec4-7d43-480b-bf1e-19356ce6912b.XML
[16:49:29] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/37/AS_CHANGE_HISTORY_20231113_fd9f03fb-c2dd-4d22-9b3b-1b13d01d7aaa.XML
[17:22:36] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/37/AS_ADDR_OBJ_DIVISION_20231113_f9b3cc4b-722a-4f49-bf0c-fa0ff7984390.XML
[17:22:36] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/37/AS_ADDR_OBJ_PARAMS_20231113_82f8559f-59d8-4fa9-8387-0f5158a3ae9f.XML
[17:25:26] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/37/AS_STEADS_PARAMS_20231113_dc65dad9-d8c6-4e6b-a87d-a9f01993e2df.XML
[17:40:41] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/37/AS_HOUSES_PARAMS_20231113_9a995474-1494-415f-a569-9aadca50e93a.XML
[18:36:40] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/37/AS_APARTMENTS_PARAMS_20231113_77b74aa6-403a-44ad-898c-beaf80693fcd.XML
[18:44:04] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/37/AS_ROOMS_PARAMS_20231113_33037a89-f752-41ad-b377-87ce77c9a225.XML
[18:44:09] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/37/AS_CARPLACES_PARAMS_20231113_bc7c94b6-e2a1-4a59-bbb0-df2adbb58654.XML
[18:44:09] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/37/AS_NORMATIVE_DOCS_20231113_286190f3-d594-4f39-9808-792ce083ecc3.XML
[18:45:05] ...  -- Data finish
[18:45:05] ...  -- Set LOGGED
[18:45:29] ... Stop process
--------------------------------------------------------
