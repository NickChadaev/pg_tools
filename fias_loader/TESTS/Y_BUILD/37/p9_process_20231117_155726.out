 save_gar_version 
------------------
               81
(1 row)

CALL
CALL
CALL
 set_gar_dt_create 
-------------------
                81
(1 row)

CALL
gar_fias.as_apartment_type: 13
gar_fias.as_addr_obj_type: 421
gar_fias.as_room_type: 3
gar_fias.as_operation_type: 27
gar_fias.as_param_type: 20
gar_fias.as_house_type: 14
gar_fias.as_add_house_type: 4
gar_fias.as_object_level: 17
gar_fias.as_norm_docs_types: 25
gar_fias.as_norm_docs_kinds: 4
gar_fias.as_reestr_objects: 792669
gar_fias.as_addr_obj: 17519
gar_fias.as_houses: 753049
gar_fias.as_steads: 91342
gar_fias.as_apartments: 332779
gar_fias.as_rooms: 6373
gar_fias.as_carplaces: 535
gar_fias.as_adm_hierarchy: 1061701
gar_fias.as_mun_hierarchy: 854005
gar_fias.as_change_history: 2624431
gar_fias.as_addr_obj_division: 16
gar_fias.as_addr_obj_params: 223728
gar_fias.as_steads_params: 1212381
gar_fias.as_houses_params: 4523519
gar_fias.as_apartments_params: 623244
gar_fias.as_rooms_params: 6526
gar_fias.as_carplaces_params: 72
gar_fias.as_normative_docs: 77085
